src/data-mgt/node/utils/transform.js

```diff
@@ -95,7 +107,8 @@ const transform = {
           } else {
             return callback({
               success: false,
-              message: "readKey unavailable, please update device details",
+              message: "readKey unavailable, this might be an external device",
```

Comment:
using readKey as a check for either device is external or internal. CONTEXT: All AirQo devices (internal) have readKeys.